specs/grafana-integration.adoc

```diff
@@ -0,0 +1,140 @@
+// vim: tw=79
+
+= Packaging tendrl grafana integration and changes
+
+Grafana is a monitoring tool works with some standard time-series db which is
+added into tendrl for visualizing tendrl metrics. It will be using graphite
+as a database. This spec file address the required changes for all the tendrl
+components to support grafana for monitoring tendrl metrics.
+
+
+== Problem description
+
+* Currently tendrl has its own dashboard for visualizating the monitoring
+data. It does not support grafana. This document will provides the details
+about the required changes in every tendrl components to support grafana
+according to package prospective.
+
+
+== Use Cases
+
+Install tendrl packages and its depdendencies to support grafana integration
+
+== Proposed change
+
+* The Makefile, spec file, service file and other build related files used
+in the tendrl app should be updated to support grafana integration.
+* All the related jobs in tendrl-copr, jenkins should also updated.
+
+
+=== Alternatives
+
+None
+
+
+=== Data model impact:
+
+None
+
+
+=== Impacted Modules:
+
+
+==== Tendrl/monitoring-integration impact:
+This is the new package added into tendrl app to dynamically configure
+grafana by using the json template with the given values or the retrieved
+values from etcd. The json will be used by the grafana service which will
+fetche the values from graphite to visualize the data. There will be a
+service which runs in the tendrl-server to perform this job.
+This package needs to be builded for tendrl.
+This package (rpm) should install all the required packages and the
+service file during installation.
+Reference Doc: https://github.com/Tendrl/specifications/pull/218
+
+
+==== Tendrl/node_agent impact:
+Node-monitoring will be integrated with node-agent. The services of
+both the components will be merged together. The changes should be
+updated into the spec file, after the merge of the pr for the
+spec: https://github.com/Tendrl/specifications/pull/219.
+Build scripts should be updated to support make-snapshot which should
+automatically update the release version as per fedora guidelines to
+integrate with centos-ci
+
+
+==== Tendrl/api impact:
+This package will be used for calling api to importing gluster and
+user identity management. The build script should be updated to support
+make-snapshot and automatic release version.
+
+
+==== Tendrl/gluster-integration impact:
+This package is now newly using belivet to fetch device details.
```

Comment:
Have taken care about adding this dependency in [spec](https://github.com/nnDarshan/specifications/commit/548be0a96fcfd19b8dc8b6f75a7e2be62676af37#diff-e83be068f21457b5b5ebb7119dc65763R99). you can drop this from this spec.